### PR TITLE
alter RSS behavior so posts don't get buried in feed readers

### PR DESCRIPTION
**Pull request by adrusi**

### CODE REVIEW

File: packages/lesswrong/components/common/SubscribeWidget.jsx [hunk 1]
```diff
@@ -0,0 +1,253 @@
+import React, { Component } from 'react';
+import _ from 'lodash';
```
Comment (Discordius) [inline packages/lesswrong/components/common/SubscribeWidget.jsx]: Remove this import

File: packages/lesswrong/components/common/SubscribeWidget.jsx [hunk 2]
```diff
@@ -0,0 +1,253 @@
...
+  'curated': 'Curated Content',
+  'community': 'Community',
+  'meta': 'Meta',
+  'pending': 'pending posts',
+  'rejected': 'rejected posts',
+  'scheduled': 'scheduled posts',
+  'all_drafts': 'all drafts',
+}
+
+class SubscribeDialog extends Component {
```
Comment (Discordius) [inline packages/lesswrong/components/common/SubscribeWidget.jsx]: Split this up into two files

File: packages/lesswrong/components/common/SubscribeWidget.jsx [hunk 3]
```diff
@@ -0,0 +1,253 @@
...
+      opacity: "0.15"
+    }
+  },
+  buttons: {
+    marginTop: "-12px",
+    marginBottom: "6px"
+  }
+});
+
+const hoursPerWeek = {
```
Comment (Discordius) [inline packages/lesswrong/components/common/SubscribeWidget.jsx]: Add comment explaining what this object stores

File: packages/lesswrong/components/common/SubscribeWidget.jsx [hunk 4]
```diff
@@ -0,0 +1,253 @@
...
+  2: "3 hours",
+  30: "2 hours",
+  45: "1 hour",
+  75: "half an hour"
+};
+
+const postsPerWeek = {
```
Comment (Discordius) [inline packages/lesswrong/components/common/SubscribeWidget.jsx]: Add comment explaining what this object stores (and how we got the data)

File: packages/lesswrong/components/common/SubscribeWidget.jsx [hunk 5]
```diff
@@ -0,0 +1,253 @@
...
+        onChange={this.handleViewChange}
+        inputProps={{ id: "subscribe-dialog-view" }}
+      >
+        <MenuItem value="curated">Curated</MenuItem>
+        <MenuItem value="frontpage">Frontpage</MenuItem>
+        <MenuItem value="community">All Posts</MenuItem>
+      </Select>
+    </FormControl>;
+
+    const closeButton = <FlatButton
```
Comment (Discordius) [inline packages/lesswrong/components/common/SubscribeWidget.jsx]: I would leave this inline. I don't think making it a mini component makes it clearer

File: packages/lesswrong/components/common/SubscribeWidget.jsx [hunk 6]
```diff
@@ -0,0 +1,253 @@
...
+  }
+
+  render() {
+    return (
+      <div className={this.props.classes.buttons}>
+        <div className={this.props.classes.subscribeButton} onClick={ () => this.openDialog("rss") }>
+          <FontIcon className={`material-icons ${this.props.classes.icon}`}>rss_feed</FontIcon>
+        </div>
+        <div className={this.props.classes.subscribeButton} onClick={ () => this.openDialog("email") }>
+          <FontIcon className={`material-icons ${this.props.classes.icon}`}>email</FontIcon>
```
Comment (Discordius) [inline packages/lesswrong/components/common/SubscribeWidget.jsx]: When you have multiple classnames, use the `classnames` package/function

File: packages/lesswrong/components/posts/PostsViews.jsx [hunk 7]
```diff
@@ -6,6 +6,7 @@ import { withRouter, Link } from 'react-router'
 import FontIcon from 'material-ui/FontIcon';
 import classnames from 'classnames';
 import Users from 'meteor/vulcan:users';
+import _ from 'lodash';
```
Comment (Discordius) [inline packages/lesswrong/components/posts/PostsViews.jsx]: Remove import

File: packages/lesswrong/components/posts/PostsViews.jsx [hunk 8]
```diff
@@ -81,7 +82,7 @@ class PostsViews extends Component {
     }
   }
 
-  handleChange = (view) => {
+  handleChange = (view, ev) => {
```
Comment (Discordius) [inline packages/lesswrong/components/posts/PostsViews.jsx]: Change to e or event (but e is better :P)

File: packages/lesswrong/components/common/SubscribeWidget.jsx [hunk 9]
```diff
@@ -0,0 +1,253 @@
...
+  subscribeButton: {
+    display: "inline-block",
+    marginLeft: "10px",
+    opacity: "0.4",
+    "&:hover": {
+      opacity: "0.15"
+    }
+  },
+  buttons: {
+    marginTop: "-12px",
```
Comment (Discordius) [inline packages/lesswrong/components/common/SubscribeWidget.jsx]: Refactor to use `theme.spacing.unit`

File: packages/lesswrong/components/common/SubscribeWidget.jsx [hunk 10]
```diff
@@ -0,0 +1,253 @@
...
+
+  handleThresholdChange = (event, threshold) => {
+    this.props.onThresholdChange(threshold);
+  }
+
+  handleViewChange = (event) => {
+    this.props.onViewChange(event.target.value);
+  }
+
+  handleClose = (event) => {
```
Comment (Discordius) [inline packages/lesswrong/components/common/SubscribeWidget.jsx]: Remove and just use inline

File: packages/lesswrong/components/common/SubscribeWidget.jsx [hunk 11]
```diff
@@ -0,0 +1,253 @@
...
+}
+
+class SubscribeDialog extends Component {
+  constructor(props) {
+    super(props);
+    this.state = { copiedRSSLink: false };
+  }
+
+  rssTerms() {
+    return { view: this.props.view + "-rss", karmaThreshold: this.props.threshold };
```
Comment (Discordius) [inline packages/lesswrong/components/common/SubscribeWidget.jsx]: Template string seems 5% better here

File: packages/lesswrong/components/comments/CommentsViews.jsx [hunk 12]
```diff
@@ -103,4 +103,7 @@ CommentsViews.defaultProps = {
 
 CommentsViews.displayName = "PostsViews";
```
Comment (Discordius) [inline packages/lesswrong/components/comments/CommentsViews.jsx]: Lol

File: packages/lesswrong/lib/collections/users/callbacks.js [hunk 13]
```diff
@@ -16,3 +16,13 @@ function updateTrustedStatus ({newDocument, vote}) {
 
 addCallback("votes.smallUpvote.async", updateTrustedStatus);
 addCallback("votes.bigUpvote.async", updateTrustedStatus);
+
+
+function maybeSendVerificationEmail (user, oldUser)
+{
+  if((""+oldUser.whenConfirmationEmailSent) != (""+user.whenConfirmationEmailSent)) {
```
Comment (Discordius) [inline packages/lesswrong/lib/collections/users/callbacks.js]: Compare with .getTime, see here: 

https://stackoverflow.com/questions/492994/compare-two-dates-with-javascript

**Comment by jimrandomh** [comment]
This branch is continued in https://github.com/LessWrong2/Lesswrong2/pull/838

File: packages/lesswrong/lib/collections/users/callbacks.js (hunk 13 above)
Comment (jimrandomh) [reply packages/lesswrong/lib/collections/users/callbacks.js]: Ended up obsoleted by switching to a different trigger, which either provides a $set.whenConfirmationEmailSent or doesn't, so it's no longer a date comparison at all.

**Pull request closed without merging**